apps/entity-service/src/entities/entities.service.ts

```diff
@@ -36,7 +36,13 @@ const ENTITY_PROCESSORS = {
 
 export type ProcessableEntity = keyof typeof ENTITY_PROCESSORS;
 export const PROCESSABLE_ENTITIES = Object.keys(ENTITY_PROCESSORS) as ProcessableEntity[];
-
+export const POLYGON_STATUSES_FILTERS = [
+  "no-polygons",
+  "submitted",
+  "approved",
+  "needs-more-information",
+  "draft"
+] as const;
```

Comment:
It's pretty nice in the code to create a type from these const arrays, and use it in the DTO:
```
export type PolygonStatusFilter = (typeof POLYGON_STATUSES_FILTERS)[number];
```

And then in the dto instead of `polygonStatus?: string`, use `polygonStatus?: PolygonStatusFilter`